code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -251,7 +251,11 @@
 	icon_state = "arm_blade"
 	item_state = "arm_blade"
 	force = 25
+
+	block_power = 50
+	canblock = TRUE
```

Comment:
Revert this actually, this doesn't need the ability to block. I missed it when I pulled back from the other armblades.